src/Uno.UI/UI/Xaml/Shapes/ArbitraryShapeBase.wasm.cs

```diff
@@ -17,5 +19,147 @@ private IDisposable BuildDrawableLayer()
 		}
 
 		private Size GetActualSize() => Size.Empty;
+
+		protected override Size MeasureOverride(Size availableSize)
+		{
+			var measurements = GetMeasurements(availableSize);
+			var desiredSize = measurements.desiredSize;
+
+			var s = base.MeasureOverride(availableSize);
+
+			return desiredSize;
+		}
+
+		protected override Size ArrangeOverride(Size finalSize)
+		{
+			var measurements = GetMeasurements(finalSize);
+
+			foreach (FrameworkElement child in GetChildren())
+			{
+				var scaleX = measurements.scaleX.ToString(CultureInfo.InvariantCulture);
+				var scaleY = measurements.scaleY.ToString(CultureInfo.InvariantCulture);
+				var translateX = measurements.translateY.ToString(CultureInfo.InvariantCulture);
+				var translateY = measurements.translateX.ToString(CultureInfo.InvariantCulture);
+
+				child.SetStyleArranged(("transform", $"scale({scaleX}, {scaleY}) translate({translateX}px, {translateY}px)"));
```

Comment:
We'll probably need to make an optimized method for that, parsing doubles is not particularly efficient.

Reply:
We need to set text for CSS anyway... so I don't see what you can optimize here.